patches/extra/ungoogled-chromium/add-flag-to-configure-extension-downloading.patch

```diff
@@ -104,13 +104,13 @@
  bool IsExtensionDownload(const download::DownloadItem& download_item);
 --- a/chrome/browser/download/download_target_determiner.cc
 +++ b/chrome/browser/download/download_target_determiner.cc
-@@ -1001,10 +1001,12 @@ DownloadConfirmationReason DownloadTarge
+@@ -1013,10 +1013,12 @@ DownloadConfirmationReason DownloadTarge
      return DownloadConfirmationReason::SAVE_AS;
  
  #if BUILDFLAG(ENABLE_EXTENSIONS)
--  // Don't prompt for extension downloads.
--  if (download_crx_util::IsExtensionDownload(*download_) ||
--      filename.MatchesExtension(extensions::kExtensionFileExtension))
+-  // Don't prompt for extension downloads if the installation site is white
```

Comment:
The new method `IsTrustedExtensionDownload` should be used here. `IsExtensionDownload` still exist, but using it would be less restrictive. I'm not sure if that's ideal.

Reply:
Thanks for the suggestion. I will go ahead and apply that.